engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/layout/pdf/hyphen/Word.java

```diff
@@ -18,6 +18,19 @@ public class Word {
 	protected int end;
 	protected String text;
 
+	private boolean keepLastSHY = true;
+
+	/**
+	 * Should a trailing SHY symbol be kept or omitted?
+	 */
+	public boolean isKeepLastSHY() {
```

Comment:
isKeepLastSoftHyphen ?

Please add @since 4.13 tag
Please add javadoc for every input and output param